app/src/main/java/com/hungngo/themusicapp/ui/search/SearchViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.hungngo.themusicapp.ui.search
+
+import androidx.lifecycle.MutableLiveData
+import com.hungngo.themusicapp.base.BaseViewModel
+import com.hungngo.themusicapp.data.TrackRepository
+import com.hungngo.themusicapp.data.model.SearchTrackResponse
+
+class SearchViewModel(private val trackRepository: TrackRepository) : BaseViewModel() {
+
+    var tracks: MutableLiveData<SearchTrackResponse> = MutableLiveData()
```

Comment:
MutableLiveData chỉ nên modify ở nội bộ class , ông khai báo thêm 1 cái LiveData get value từ cái MutableLiveData này
```suggestion
    private val tracks = mutableLiveData<SearchTrackResponse>()
```